snapcraft.yaml

```diff
@@ -0,0 +1,113 @@
+name: desmume-emulator
+version: 0.9.13
+summary: DeSmuME is a Nintendo DS Emulator
+description: |
+  An open-source Nintendo DS Emulators.
+adopt-info: desmume-emulator
+grade: stable
+confinement: strict
+base: core22
+compression: lzo
+parts:
+  desmume-emulator:
+    source: https://github.com/TASEmulators/desmume/archive/refs/tags/release_0_9_13.tar.gz
+    plugin: nil
+    override-build: |
+      LDFLAGS="" CFLAGS="" FFLAGS="" FCFLAGS="" DEB_CFLAGS_SET="" DEB_CXXFLAGS_SET="" DEB_FFLAGS_SET="" DEB_FCFLAGS_SET="" DEB_LDFLAGS_SET="" meson setup --prefix=${CRAFT_PART_INSTALL}/usr desmume/src/frontend/posix/build desmume/src/frontend/posix
```

Comment:
should probably set CFLAGS to -O2 or -O3, else desmume will be very slow

Reply:
ok I suspect that there are still a few things to correct